ui/packages/consul-ui/app/components/filter-bar/skin.scss

```diff
@@ -1,14 +1,14 @@
 .filter-bar {
   & {
-    background-color: rgb(var(--tone-gray-010));
+    background-color: var(--token-color-foreground-high-contrast);
     border-bottom: var(--decor-border-100);
```

Comment:
Looks like there is an extra bottom border showing on the peers filter bar... 
note: this is currently like this in production so not because of color updates, so happy to add a separate ticket to remove the additional boarder here if we want to tackle separately)
![image](https://user-images.githubusercontent.com/91899352/221216364-115960a0-9626-4857-a18e-04d4d22d12c1.png)
![image](https://user-images.githubusercontent.com/91899352/221216438-77380687-108a-4fa3-a681-d2f7cfa38cd6.png)

Reply:
Done ✅